apollo-federation/src/schema/validators/cache_tag.rs

```diff
@@ -185,6 +178,12 @@ fn validate_args_selection(
                 .ok_or_else(|| CacheTagValidationError::CacheTagInvalidFormat {
                     message: format!("unknown field \"{name}\""),
                 })?;
+        let is_nullable = matches!(field, Type::Named(_) | Type::List(_));
+        if is_nullable {
```

Comment:
Actually, there is `is_non_null` method.
```suggestion
        if !field.is_non_null() {
```